libexec/mcd/mcd-flips

```diff
@@ -39,16 +40,16 @@ case $1 in
       done
   };;
   beg) {
-    mcd --to-wad "$(seth call "$FLIPPER" 'beg()(uint)')"
+    mcd --to-wad "$(seth call "$FLIPPER" 'beg()')"
   };;
   ttl) {
-    seth --to-dec "$(seth call "$FLIPPER" 'ttl()(uint48)')"
+    seth --to-dec "$(seth call "$FLIPPER" 'ttl()')"
```

Comment:
Same previous file, you could directly define the return data and avoid the `seth --to-dec`, excepting we would like to do it compatible for old and new seth versions.